Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js

```diff
@@ -0,0 +1,564 @@
+import {
+  AttributeType,
+  combine,
+  ComponentDatatype,
+  GltfLoader,
+  GeometryPipelineStage,
+  Resource,
+  ResourceCache,
+  ShaderBuilder,
+  VertexAttributeSemantic,
+} from "../../../Source/Cesium.js";
+import createScene from "../../createScene.js";
+import waitForLoaderProcess from "../../waitForLoaderProcess.js";
+
+describe(
+  "Scene/GeometryPipelineStage",
+  function () {
+    var positionOnlyPrimitive = {
+      attributes: [
+        {
+          semantic: VertexAttributeSemantic.POSITION,
+          buffer: new Float32Array([0, 1, 2, 3, 4, 5]).buffer,
+          type: AttributeType.VEC3,
+          componentDatatype: ComponentDatatype.FLOAT,
+        },
+      ],
+    };
+
+    var customAttributePrimitive = {
+      attributes: [
+        {
+          semantic: VertexAttributeSemantic.POSITION,
+          buffer: new Float32Array([0, 1, 2, 3, 4, 5]).buffer,
+          type: AttributeType.VEC3,
+          componentDatatype: ComponentDatatype.FLOAT,
+        },
+        {
+          name: "_TEMPERATURE",
+          buffer: new Uint32Array([0, 1, 2, 3, 4, 5]).buffer,
+          type: AttributeType.VEC2,
+          componentDatatype: ComponentDatatype.UNSIGNED_SHORT,
+        },
+      ],
+    };
+
+    var boomBoxSpecularGlossiness =
+      "./Data/Models/GltfLoader/BoomBox/glTF-pbrSpecularGlossiness/BoomBox.gltf";
+    var boxTextured =
+      "./Data/Models/GltfLoader/BoxTextured/glTF-Binary/BoxTextured.glb";
+    var boxVertexColors =
+      "./Data/Models/GltfLoader/BoxVertexColors/glTF/BoxVertexColors.gltf";
+    var microcosm = "./Data/Models/GltfLoader/Microcosm/glTF/microcosm.gltf";
+    var weather = "./Data/Models/GltfLoader/Weather/glTF/weather.gltf";
+    var buildingsMetadata =
+      "./Data/Models/GltfLoader/BuildingsMetadata/glTF/buildings-metadata.gltf";
+
+    var scene;
+    var gltfLoaders = [];
+
+    beforeAll(function () {
+      scene = createScene();
+    });
+
+    afterAll(function () {
```

Comment:
@sanjeetsuhag add an afterEach() function that calls `ResourceCache.clearForSpecs()`, otherwise you'll sometimes get failures due to double-unloading things from the cache.

See `ResourceCacheSpec` for an example.

Reply:
Er wait totally didn't see the afterEach() below this comment

Reply:
But check in other files, `ModelExperimentalSpec` is one case that probably needs the afterEach(), perhaps others.